app/src/main/java/cs465/illinois/edu/dogonthequad/Meetup.java

```diff
@@ -0,0 +1,27 @@
+package cs465.illinois.edu.dogonthequad;
+
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.UUID;
+
+public class Meetup {
+
+    public double mLocationLatitude = 0.0;
+    public double mLocationLongitude = 0.0;
+
+    public int mLengthMinutes = 0;
```

Comment:
Didn't we decide to change to a end time instead? Also, remember to use `java.util.Date` objects! (And I think there's time difference objects too.)

Reply:
It is a Java.util.Date object?

Reply:
Perhaps it's something different. But whenever possible, use some sort of timekeeping object.

Reply:
Yes, but only if we stick with setting the meetup length instead of an end time. But our user tests indicated that selecting an end time would be easier, so I'd recommend that.

Reply:
Didn't realize that's what you meant/that's what we committed to but sure, fixed. Obviously all of this structure is subject to change, and will likely change as we implement stuff so there's no point in debating it now. Let's just get it merged in and then we can change things as we go.